CHANGELOG.md

```diff
@@ -7,6 +7,12 @@
 * `RegistryConfigs.Builder#addConfig` does not ignore null `RegistryAuth` values anymore
 * `HostConfig.Builder#appendBinds*` methods removed
 * `HostConfig.Builder#binds(final Bind... binds)` method does not ignore null binds anymore
+* `Immutable*` types removed from public getters and builders
+* `EventStream` now implements `java.util.Iterator<Event>`
+* `EventStream#close` and `LogStream#close` can now throw `IOException`
+* `LogStream#attach(OutputStream, OutputStream, boolean)` method removed. Methods should
+not close streams they do not manage.
+* `LogStream#attach(OutputStream, OutputStream)` can now throw `IOException`
```

Comment:
Consider adding a bullet for "removed dependency on Guava, which resulted in some of the breaking changes"

Reply:
This would be added along with the commits just before releasing 3.0.0. I 've added these bullets here so I won't forget them :)